movies/movies/TableView/AboutMoviesPresenter.swift

```diff
@@ -0,0 +1,69 @@
+// AboutMoviesPresenter.swift
+// Copyright © RoadMap. All rights reserved.
+
+import Alamofire
+import Foundation
+import UIKit
+
+//MARK: - Protocol
+
+protocol AboutMoviesPresenterProtocol: AnyObject {
+    var aboutMovie: Result? { get set }
+    init(view: AboutMoviesViewProtocol, networkService: NetworkServiceProtocol)
+    func fetchDataAboutMovie(cell: AboutMovieTableViewCell, indexPath: IndexPath, idMovie: Int)
+    func getImage(cell: AboutMovieTableViewCell)
+}
+
+protocol AboutMoviesViewProtocol: AnyObject {
+    func reloadData()
+}
+
+final class AboutMoviesPresenter: AboutMoviesPresenterProtocol {
+    
+    //MARK: - Public Properties
+    
+    var networkService: NetworkServiceProtocol!
+    var aboutMovie: Result?
+    var posterPath = String()
+    
+    //MARK: - Private Properties
+    
+    private weak var view: AboutMoviesViewProtocol!
+    
+    //MARK: - Intial
+    required init(view: AboutMoviesViewProtocol, networkService: NetworkServiceProtocol) {
+        self.networkService = networkService
+        self.view = view
+    }
+    
+    //MARK: - Public Properties
+    
+    func presenterSetup(view: AboutMoviesViewProtocol) {
+        self.view = view
+    }
+    
+    func fetchDataAboutMovie(cell: AboutMovieTableViewCell, indexPath: IndexPath, idMovie: Int) {
```

Comment:
```suggestion
    func fetchDataAboutMovie(, idMovie: Int) {
```
это вью cell: AboutMovieTableViewCell, indexPath: IndexPath презентер не должен ничего о них знать